src/qibolab/_core/platform/load.py

```diff
@@ -38,7 +39,7 @@ def _search(name: str, paths: list[Path]) -> Path:
     )
 
 
-def _load(platform: Path) -> Platform:
+def _load(platform: Path) -> Union[Platform, Hardware]:
```

Comment:
We should open an issue for 0.3 to drop the support for `Platform`.

Of course, it's just to schedule it, since it's clearly far in the future (and also not a big deal - but let's keep things clean as possible).

Reply:
#1130